po/.gitignore

```diff
@@ -1,3 +0,0 @@
-Makefile.in.in
```

Comment:
Ugh, admittedly I wasn't even aware that we had gitignore files in subdirectories. By all means let's please kill them. Thanks!